static/js/init.js

```diff
@@ -3,6 +3,7 @@
 /* global require, window, document */
 
 var $ = require('jquery');
+var keyboard = require('keyboardjs');
```

Comment:
If we don't have trouble with our assets being too heavy, then I'm :+1: to including a library for convenience.